package.json

```diff
@@ -102,7 +102,7 @@
     "axe-core": "4.6.3",
     "comment-json": "4.2.3",
     "dom-autoscroller": "2.3.4",
-    "dompurify": "3.0.1",
```

Comment:
My updates merged - so I believe there are merge conflicts here with that. Sorry!